### PR TITLE
Issue processing

### PR DESCRIPTION
**Pull request by zuckung**
A new workflow starts when an issue is created. 
It runs a python script which reads the body content of that issue and decides then if that issue is a plugin submission or not.
If it is a new plugin, it creates a txt file and puts it into res/pluginlist/ .
Then res/pluginlist/ is pushed to a new created branch with unique name.
And a pull request of that branch gets created.

### CODE REVIEW

**Comment by zuckung** [comment]
And you have to allow the workflow action to create pull requests.

settings > actions > general > allow github action to create and approve pull requests

**Comment by zuckung** [comment]
Here is some real data of a new plugin, for a test run.

Eternal Shuttle
mOctave
https://github.com/mOctave/eternal-shuttle
https://github.com/mOctave/eternal-shuttle/archive/refs/heads/main.zip
Story
Work In Progress
An Endless Sky plugin about the end of humanity and fighting off an alien invasion in a Shuttle.

File: res/src/onissue.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+		if line.find("pluginurl") == 0:	# i.e. pluginurl = https://github.com/zuckung/test3/tree/main/Working/All%20Plugins/
+			pluginurl = line.split(" = ")[1]
+		if line.find("assetfullpath") == 0:	# i.e. assetfullpath = https://github.com/zuckung/test3/releases/download/Latest/
+			assetfullpath = line.split(" = ")[1]
+		if line.find("newsamount") == 0:	# i.e. newsamount = 20
+			newsamount = line.split(" = ")[1]
+
+def format_content():
+	splitted = content.split('\n')
+	if splitted[0] == '### Pluginname\r':
```
Comment (Hecter94) [inline res/src/onissue.py:28]:
```suggestion
	if splitted[0] == '### Name\r':
```
I changed this in the template to be just "Name", not "Pluginname"

File: res/template.txt [hunk 2]
```diff
@@ -93,7 +93,7 @@ Some plugins are still missing important information, such as the author, descri
 
 ## Add your plugin
 
-Do you want your plugin here? Fork this repository, upload your plugin, and make a Pull Request. Besides that, you can post it in the discussion area.
+Do you want your plugin here? Fork this repository, upload your plugin, and make a Pull Request. Besides that, you can post it in the discussion area, or better create a plugin submission issue.
```
Comment (Hecter94) [inline res/template.txt:96]:
```suggestion
Do you want your plugin here? Create a Pull Request or open a new [issue](https://github.com/Hecter94/EndlessSky-PluginArchive/issues/new?assignees=&labels=&projects=&template=plugin-submission.yaml)
```

**Review by Hecter94 — CHANGES_REQUESTED** [review]

**Pull request merged**